util/notify/README.org

```diff
@@ -21,6 +21,12 @@ In order to get your own notifications handler you can define on function
 
 ** Dependencies
 
+To install dependencies (if you are using quicklisp):
```

Comment:
A shorter way to install the dependencies is to  load the system itself, ie. `(ql:quickload "notify")`. btw one can pass in a list of system names to quickload

Reply:
Thanks for the pointers! :+1: